src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -116,7 +116,8 @@ public Object invokeMethod(String name, Object args) {
         FlowNode an;
 
         // TODO: generalize the notion of Step taking over the FlowNode creation.
```

Comment:
@abayer when filing an issue corresponding to a `TODO` comment, please either delete the comment, or update it to refer to the issue ID.